DepotDownloader/Program.cs

```diff
@@ -159,8 +182,17 @@ static bool InitializeSteam( string username, string password )
                 do
                 {
                     Console.Write( "Enter account password for \"{0}\": ", username );
-                    password = Util.ReadPassword();
-                    Console.WriteLine();
+                    // Avoid console echoing of password
+                    try
+                    {
+                        password = Util.ReadPassword();
+                        Console.WriteLine();
+                    }
+                    // Fallback to support redirected input

```

Comment:
Why not just use `Console.IsInputRedirected`?

Reply:
I didn't know it existed, thanks! I didn't see your comments until I pushed a new revision below, but I'll look at it when I have time again.